packages/jira-adapter/src/filters/script_runner/workflow_scripts.ts

```diff
@@ -0,0 +1,182 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { gzip, gunzip } from 'zlib'
+import { logger } from '@salto-io/logging'
+import { collections } from '@salto-io/lowerdash'
+import { createSchemeGuard, safeJsonStringify } from '@salto-io/adapter-utils'
+import Joi from 'joi'
+import { isInstanceElement, Element, isInstanceChange, isAdditionOrModificationChange, getChangeData, InstanceElement, Value } from '@salto-io/adapter-api'
+import { FilterCreator } from '../../filter'
+import { WORKFLOW_TYPE_NAME } from '../../constants'
+
+const { awu } = collections.asynciterable
+const { makeArray } = collections.array
+const log = logger(module)
+const scriptRunnerPostFunctionType = 'com.onresolve.jira.groovy.groovyrunner__script-postfunction'
```

Comment:
super nit: consts convention:
```suggestion
const SCRIPT_RUNNER_POST_FUNCTION_TYPE = 'com.onresolve.jira.groovy.groovyrunner__script-postfunction'
```

Reply:
not a nit actually :)